tests/bin/test_busd_smoke.py

```
@@ -1,71 +1,71 @@
+        }
+        
+        msg_file = Path(test_dir, "mbox/pmai/in/test.json")
+        msg_file.write_text(json.dumps(test_msg))
+        
+        print(f"Created test message: {msg_file}")
+        
+        # mailbox処理を一度実行
+        try:
+            busd.process_mailbox_once()
+            print("✓ process_mailbox_once executed")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)